apps/website/src/pages/show-and-tell/index.tsx

```diff
@@ -391,6 +401,9 @@ const ShowAndTellIndexPage: NextPage<ShowAndTellPageProps> = ({
           onKeyDown={handleArrowKeys}
           tabIndex={-1}
         >
+          <h1 className="text-center text-3xl italic">
+            {totalPostsCount} posts by {usersCount} users
```

Comment:
```suggestion
            {totalPostsCount.toLocaleString()} posts by {usersCount.toLocaleString()} users
```
We should locale format both the numbers here, so they get thousands separators etc.

Reply:
I think that would break SSR for different locale.

Reply:
What's the recommended way to do that, that doesn't break SSR? useMemo them?

Reply:
For the pages router and SSR I'd say `useState` + `useEffect`, yeah. I don't think you can use client components with `"use client"` in the pages router. It's kinda a special case since we do not have the language/locale in the router or server-side at all.

Something like:

```typescript
function useLocale() {
  const [locale, setLocale] = useState('en-US'); // default for initial render
  useEffect(() => {
    setLocale(undefined); // use browser default locale
  }, []);
  return locale;
}

const locale = useLocale();
value.toLocaleString(locale);
```

Reply:
or honestly just use US formatting